src/server.ts

```diff
@@ -1,11 +1,12 @@
 import { PORT } from './config';
 import app from './app';
+import NodeEnv from './shared/constants/NodeEnv';
```

Comment:
we wouldn't want to create a new object - instead we would want to update the global type of `process.env.NODE_ENV` so that when we attempt to do something like `process.env.NODE_ENV === 'build'` we don't get a TS error

Reply:
That'd cause an error just as I mentioned on Slack, extending the interface ends in errors.

Reply:
@Ebugo have you tried this? https://stackoverflow.com/a/53981706